File: backend/src/main/java/com/revnu/backend/features/settings/dto/RestaurantProfileResponse.java
package com.revnu.backend.features.settings.dto;

import java.util.UUID;

public record RestaurantProfileResponse(
        UUID id,
        String restaurantName,
        String physicalAddress,
        String openingHours,
        String closingHours,
        UUID logoFileId
        ) {

}

File: mobile/app/src/main/java/com/revnu/mobile/features/settings/model/RestaurantProfileResponse.kt
package com.revnu.mobile.features.settings.model

import com.google.gson.annotations.SerializedName

data class RestaurantProfileResponse(
    @SerializedName("id") val id: String,
    @SerializedName("restaurantName") val restaurantName: String,
    @SerializedName("physicalAddress") val physicalAddress: String,
    @SerializedName("openingHours") val openingHours: String,
    @SerializedName("closingHours") val closingHours: String,
    @SerializedName("logoFileId") val logoFileId: String?
)
